Lib/fontTools/pens/t2CharStringPen.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2009 Type Supply LLC
+# Author: Tal Lemming
+
+
+from __future__ import print_function, division, absolute_import
```

Comment:
As you import `fontTools.misc.py23` now I think it does the future imports by itself.